scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontendState.java

```diff
@@ -128,19 +128,26 @@
      * and job/user association.
      */
     private void recover(SchedulerStateImpl sState) {
+        Vector<JobState> pendingJobs = sState.getPendingJobs();
+        Vector<JobState> runningJobs = sState.getRunningJobs();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace the synchronized class "Vector" by an unsynchronized one such as "ArrayList" or "LinkedList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1149)